src/main/java/com/exactpro/th2/common/schema/message/MessageRouter.java

```diff
@@ -61,7 +61,12 @@ default void init(@NotNull MessageRouterContext context, @NotNull MessageRouter<
     ExclusiveSubscriberMonitor subscribeExclusive(MessageListener<T> callback);
 
     /**
-     * Listen <b>ONE</b> RabbitMQ queue by intersection schemas queues attributes
+     * Listen <b>ONE</b> RabbitMQ queue by intersection schemas queues attributes.
+     * Restrictions:
+     * You can create only one subscription to th2 pin using any subscribe* functions. This method throws RuntimeException if the th2 pin is matched by passed attributes already has active subscription.
```

Comment:
I think information about throwing an exception should be moved to a corresponding block `@throws`